roles/sfia/delivery_manager.md

```diff
@@ -0,0 +1,89 @@
+# SFIA Role Guidance: Delivery Manager 
+
+[SFIA Level 4: Follow](https://sfia-online.org/en/sfia-7/responsibilities/level-4)
+
+[next &raquo;](senior_delivery_manager.md)
+
+## Required competency for role
+
+### Autonomy
```

Comment:
The various attributes listed have a mix of full stop and no full stop - suggest making them consistent (either way)